Ginger/Ginger/UserControlsLib/ImageMakerLib/ImageMakerControl.xaml.cs

```diff
@@ -521,6 +524,18 @@ private void SetImage()
                 case eImageType.Camera:
                     SetAsFontAwesomeIcon(FontAwesomeIcon.Camera);
                     break;
+                case eImageType.ExcelExport:
+                    SetAsFontAwesomeIcon(FontAwesomeIcon.FileExcelOutline);
+                    break;
```

Comment:
Change the eImageType.ExcelExport to eImageType.ExcelFile so it will be used for other cases we will need Excel icon also for not Export operations

Reply:
Done